tests/shared/src/test/scala-3/scala/meta/tests/quasiquotes/Scala3SpecificSuccessSuite.scala

```diff
@@ -13,22 +13,12 @@ class Scala3SpecificSuccessSuite extends TreeSuiteBase {
         class $name:
           val a: Int = 1
       """
-    assertEquals(
-      compileErrors("""inlinedMultiline(Type.Name("AAA"))"""),
-      s"""error: double quotes are not allowed in single-line quasiquotes
-         |package quasiquotes
-         |^""".stripMargin
-    )
+    inlinedMultiline(Type.Name("AAA"))
```

Comment:
if there's no `assrtNoThrow`, i'd at least add a comment that _that's_ what we are testing.